app/views/records/_filter_content.html.haml

```diff
@@ -7,4 +7,4 @@
   \# Supplejack was created by DigitalNZ at the National Library of NZ and the Department of Internal Affairs.
   \# http://digitalnz.org/supplejack
 - filter_facets.each_with_index do |facet, index|
-  = render 'filter_tab', facet: facet[0], facet_values: facet[1], index: index if facet[1].size > 1
```

Comment:
This should be changed back to using `facet` and `facet_values`, the line above this should extract them from the `facet` area like so `name, values = facet`